clang/include/clang/Driver/Options.td

```diff
@@ -2788,6 +2788,11 @@ def fintelfpga : Flag<["-"], "fintelfpga">, Group<f_Group>,
   Flags<[CC1Option, CoreOption]>, HelpText<"Perform ahead of time compilation for FPGA">;
 def fsycl_device_only : Flag<["-"], "fsycl-device-only">, Flags<[CoreOption]>,
   HelpText<"Compile SYCL kernels for device">;
+defm sycl_esimd_force_stateless_mem : BoolFOption<"sycl-esimd-force-stateless-mem",
+    LangOpts<"SYCLESIMDForceStatelessMem">, DefaultFalse,
+    PosFlag<SetTrue, [CC1Option, CoreOption], "Convert accesses via accessors to USM memory accesses in ESIMD kernels">,
+    NegFlag<SetFalse>>;
```

Comment:
Should also be a `CoreOption` to be available on Windows command lines

Reply:
Mike, Thank you for reviewing the PR. Can you explain what you mean, or maybe add a 'suggestion'?
Isn't it already a CoreOption (see the line 2793)?

Reply:
The positive variant of the option is a `CoreOption` but the negative isn't.  I don't think the Flags from the Pos side are also applied to the Neg side unless `BothFlags` is used to do so.

Reply:
I added the BothFlags to address this comment. Thank you!